### PR TITLE
feat(accountlib): BGA-656 Implement KeyPair Generation and Unit Tests

### PR DESCRIPTION
**Pull request by smaria95**
BGA-656 KeyPair Generation Implemented along with Unit Tests.

**Notes**:
- @bitgo/statics version has been updated to 6.1.0-rc.1 since it's required for unit tests in this task. Previous version was v6.0.0.
- Build is failing in drone due to a version issue with secp256k1 lib between BitGoJs and casper-client.

### CODE REVIEW

File: modules/account-lib/package.json
Comment (tylerlevine) [inline modules/account-lib/package.json:36]: Please don't modify the versions of packages which are inside this monorepo (that's `@bitgo/statics`, `bitgo` or `@bitgo/account-lib` for now). These versions are managed by the monorepo management tool lerna during release, and modifying it outside a release branch will create merge conflicts when I release new versions.

**Review by tylerlevine — CHANGES_REQUESTED** [review]

File: modules/account-lib/package.json
Comment (smaria95) [reply modules/account-lib/package.json:36]: Hi  @tylerlevine.

We need statics version 6.1.0 not only to run the tests but also to being able to import casper-client as dependency.
How should we proceed with this if we cannot upgrade this lib until release?

Comment (tylerlevine) [reply modules/account-lib/package.json:36]: When you are developing in this monorepo, the dependencies for packages within the monorepo are not downloaded and installed from npm. Instead they are symlinked to the version in the monorepo. If you have already made the changes you need in `statics`, then there is no need to update this version and you will get the version that currently exists in `master`.

**Review by CapnMigraine — CHANGES_REQUESTED** [review]
Depends on https://github.com/BitGo/BitGoJS/pull/975

**Comment by tylerlevine** [comment]
#975 is merged, please rebase on top of latest master

**Review by tylerlevine — APPROVED** [review]
lgtm, should be good once @CapnMigraine approves

**Review by CapnMigraine — APPROVED** [review]
lgtm

**Pull request merged**
